src/components/standalone/security/threat_shield/BlocklistTab.vue

```diff
@@ -0,0 +1,180 @@
+<!--
+  Copyright (C) 2023 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { ubusCall } from '@/lib/standalone/ubus'
+import { useUciPendingChangesStore } from '@/stores/standalone/uciPendingChanges'
+import { NeBadge, NeButton, NeSkeleton, NeInlineNotification } from '@nethesis/vue-components'
+import { getAxiosErrorMessage, NeTextInput, NeEmptyState } from '@nethserver/vue-tailwind-lib'
+import { onMounted } from 'vue'
+import { computed } from 'vue'
+import { ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import BlocklistTable from './BlocklistTable.vue'
+import { useRouter } from 'vue-router'
+import { getStandaloneRoutePrefix } from '@/lib/router'
+
+export type Blocklist = {
+  name: string
+  type: 'community' | 'enterprise' | 'unknown'
+  enabled: boolean
+  confidence: number
+  description: string
+}
+
+const { t } = useI18n()
+const uciChangesStore = useUciPendingChangesStore()
+const router = useRouter()
+
+const error = ref({
+  notificationTitle: '',
+  notificationDescription: '',
+  notificationDetails: ''
+})
+const fetchError = ref(false)
+const loading = ref(true)
+const isTogglingBlocklistEnabled = ref(false)
+const blocklists = ref<Blocklist[]>([])
+const filter = ref('')
+const isThreatShieldEnabled = ref(false)
+
+const filteredBlocklists = computed(() => {
+  return filter.value
+    ? blocklists.value.filter((x) => x.name.includes(filter.value))
+    : blocklists.value
+})
+const isEnterprise = computed(() => {
+  return blocklists.value.some((x) => x.type === 'enterprise')
+})
+
+function cleanError() {
+  error.value = {
+    notificationTitle: '',
+    notificationDescription: '',
+    notificationDetails: ''
+  }
+}
+
+async function fetchBlocklists() {
+  cleanError()
+
+  try {
+    loading.value = true
+    blocklists.value = (await ubusCall('ns.threatshield', 'list-blocklist')).data.data
+    isThreatShieldEnabled.value = (
+      await ubusCall('ns.threatshield', 'list-settings')
+    ).data.data.enabled
+  } catch (err: any) {
+    error.value.notificationTitle = t('error.cannot_retrieve_blocklists')
+    error.value.notificationDescription = t(getAxiosErrorMessage(err))
+    error.value.notificationDetails = err.toString()
+    fetchError.value = true
```

Comment:
It seems like this variable is never reset to false, maybe we should do it before the `try`